tests/main/interfaces-framebuffer/task.yaml

```diff
@@ -18,8 +18,13 @@ execute: |
     snap interfaces -i framebuffer | MATCH '^- +test-snapd-framebuffer:framebuffer'
 
     if [ ! -e /dev/fb0 ]; then
-        echo "Framebuffer not available on /dev/fb0"
-        exit 1
+        # ensure this test runs on at least one system
+        if [[ "$SPREAD_SYSTEM" = ubuntu-core-16-arm-* ]]; then
+            echo "ubuntu-core-16-arm-32 must have a framebuffer"
+            exit 1
+        fi
+        echo "SKIP: Framebuffer not available on /dev/fb0"
+        exit 0
```

Comment:
Instead of skipping the test could blacklist `ubuntu-18.10-*`?

Reply:
This will also fail on ubuntu-16.04-amd64 once it hits the autopkgtest environment there. The current approach of a blacklist is not ideal for this because we will always play catchup, when we add a new environment into the tests the test is likely to fail.